eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -345,6 +345,18 @@ stages:
         targetOS: windows
         targetArchitecture: x64
 
+    - template: ../jobs/vmr-build.yml
+      parameters:
+        buildName: iOSSimulator_Shortstack
+        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+        vmrBranch: ${{ variables.VmrBranch }}
+        architecture: arm64
+        pool:
+          vmImage: ${{ variables.defaultPoolNameMac }}
+        container: ''
```

Comment:
Why empty?

Reply:
Empty was being used on Windows & Mac in Alex's CI PR, 28012e74ff. I generally have a policy of "if it looks weird it probably ended up weird for a good reason"

Reply:
@akoeplinger PTAL

Reply:
I also copied it from existing code :)

I agree we should clean it up, I'll take a look.

Reply:
https://github.com/dotnet/installer/pull/18590